config/src/generator.rs

```diff
@@ -52,3 +48,22 @@ pub fn validator_swarm_for_testing(nodes: usize) -> ValidatorSwarm {
     config.test = Some(TestConfig::open_module());
     validator_swarm(&NodeConfig::default(), nodes, [1u8; 32], true)
 }
+
+fn build_seed_peers(config: &NodeConfig) -> SeedPeersConfig {
```

Comment:
I recommend we just delete this, there is no test cases that need seed peers from generator -- it is purely a feature intended /only/ for swarms.

Reply:
smoke tests appear to rely on `generator` setting the seed peers